game.go

```diff
@@ -112,7 +112,6 @@ type Spriter interface {
 }
 type Gamer interface {
 	initGame(sprites []Spriter) *Game
-	getGame() *Game
```

Comment:
For now we need to maintain the same interface `Gamer` in ispx to make it work, see details in https://github.com/goplus/builder/blob/35c6e0883c5c66b29eaeed2bf99d0731f06774f9/tools/ispx/main.go#L74-L76

It is better to keep `Gamer` as simple as possible.